README.md

```diff
@@ -194,6 +194,8 @@ list.files(
 
 </details>
 
+## Contributors
```

Comment:
Hmm not really clear why this is happening (this is from all contributors and not you). Can you remove the README updates as this should now only trigger as PR anyways on main (so we can handle issues like that there)